docs/9-roadmap.md

```diff
@@ -4,19 +4,48 @@ sidebar_position: 9
 
 # Roadmap
 
-:::caution Hard Hat Area
-This page is still being developed. More content will be added soon!
-:::
+The project roadmap for Zarf is hosted on a [GitHub Project Board](https://github.com/orgs/defenseunicorns/projects/1) that tracks the issues the Zarf team is working along with future work we are prioritizing.
+
+If you would like to add bug reports or feature requests, please [add an issue](https://github.com/defenseunicorns/zarf/issues) to the GitHub repository under the appropriate template.  If you have a more general question about a feature, feel free to ask the team in the [Zarf Kubernetes Slack Channel](https://kubernetes.slack.com/archives/C03B6BJAUJ3).
+
+We also accept contributions from the community (regardless of where a particular bug or feature is on the roadmap), so feel free to read our [contributing guidelines](./12-contribute-to-zarf/1-contributor-guide.md) and [submit a PR](https://github.com/defenseunicorns/zarf/pulls)!  You can also ask any development related questions in the [Zarf Dev Kubernetes Slack Channel](https://kubernetes.slack.com/archives/C03BP9Z3CMA).
+
+## Feature Stability
+
+Zarf has three levels of feature stability that are defined below but generally follow existing Kubernetes conventions.  These communicate how a feature may change in the future as well as expectations for *ilities like documentation.
```

Comment:
you finally got `*ilities` in the docs

Reply:
It is a real word I swear! https://en.wiktionary.org/wiki/ility